project/Http4sPlugin.scala

```diff
@@ -341,6 +341,7 @@ object Http4sPlugin extends AutoPlugin {
   lazy val prometheusHotspot                = "io.prometheus"          %  "simpleclient_hotspot"      % prometheusClient.revision
   lazy val parboiled                        = "org.http4s"             %% "parboiled"                 % "1.0.1"
   lazy val quasiquotes                      = "org.scalamacros"        %% "quasiquotes"               % "2.1.0"
+  lazy val reactiveStreamsFlowAdapters      = "org.reactivestreams"    %  "reactive-streams-flow-adapters" % "1.0.2" // TODO formatting
```

Comment:
huge diff < aligned code?

Reply:
Ugh.  Leave it for now.  Let's get 0.18 EOL'ed with the release of 0.20, and we can at least reduce the pain it will cause.